packages/host/config/schema/1735668047598_schema.sql

```diff
@@ -23,6 +23,7 @@
    fitted_html BLOB,
    display_names BLOB,
    resource_created_at,
+   icon_html TEXT,
```

Comment:
The internal convention we have is to delete the old sql schema file, keeping only the latest one